# this message is sent to update some value in the clients or server:
# current values, default values, minimums, maximums.

Header header
ParamValue[] entries